pkg/agent/controller/l7flowexporter/l7_flow_export_controller.go

```diff
@@ -111,6 +112,7 @@ func NewL7FlowExporterController(
 		},
 		resyncPeriod,
 	)
+	startSuricataOnceFn = l7Reconciler.StartSuricataOnce
```

Comment:
you could replace `l7Reconciler *l7engine.Reconciler` with `suricataStartFunc func()`, and store it as a member variable, then it's easier to pass a fake implementation in unit test without introducing the global variable which is bad for encapsulation.